<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="0" tests="1" failures="1">
  <testsuite name="Root Suite" timestamp="2021-10-16T20:24:46" tests="0" file="cypress/integration/website_spec.js" failures="0" time="0">
  </testsuite>
  <testsuite name="Website Test" timestamp="2021-10-16T20:24:46" tests="1" failures="1" time="0">
    <testcase name="Website Test &quot;before all&quot; hook for &quot;should contain website title&quot;" time="0" classname="&quot;before all&quot; hook for &quot;should contain website title&quot;">
      <failure message="cy.visit() failed trying to load:

http://localhost:3000/

We attempted to make an http request to this URL but the request failed without a response.

We received this error at the network level:

  &gt; Error: connect ECONNREFUSED 127.0.0.1:3000

Common situations why this would fail:
  - you don&apos;t have internet access
  - you forgot to run / boot your web server
  - your web server isn&apos;t accessible
  - you have weird network configuration settings on your computer

The stack trace for this error is:

Error: connect ECONNREFUSED 127.0.0.1:3000
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1056:14)


Because this error occurred during a &apos;before all&apos; hook we are skipping the remaining tests in the current suite: &apos;Website Test&apos;" type="CypressError"><![CDATA[CypressError: cy.visit() failed trying to load:

http://localhost:3000/

We attempted to make an http request to this URL but the request failed without a response.

We received this error at the network level:

  > Error: connect ECONNREFUSED 127.0.0.1:3000

Common situations why this would fail:
  - you don't have internet access
  - you forgot to run / boot your web server
  - your web server isn't accessible
  - you have weird network configuration settings on your computer

The stack trace for this error is:

Error: connect ECONNREFUSED 127.0.0.1:3000
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1056:14)


Because this error occurred during a 'before all' hook we are skipping the remaining tests in the current suite: 'Website Test'
    at Object.cypressErr (http://localhost:62304/__cypress/runner/cypress_runner.js:86207:11)
    at Object.throwErr (http://localhost:62304/__cypress/runner/cypress_runner.js:86162:18)
    at Object.throwErrByPath (http://localhost:62304/__cypress/runner/cypress_runner.js:86194:17)
    at http://localhost:62304/__cypress/runner/cypress_runner.js:71500:31
    at visitFailedByErr (http://localhost:62304/__cypress/runner/cypress_runner.js:71006:12)
    at http://localhost:62304/__cypress/runner/cypress_runner.js:71499:22
    at tryCatcher (http://localhost:62304/__cypress/runner/cypress_runner.js:120203:23)
    at Promise._settlePromiseFromHandler (http://localhost:62304/__cypress/runner/cypress_runner.js:118139:31)
    at Promise._settlePromise (http://localhost:62304/__cypress/runner/cypress_runner.js:118196:18)
    at Promise._settlePromise0 (http://localhost:62304/__cypress/runner/cypress_runner.js:118241:10)
    at Promise._settlePromises (http://localhost:62304/__cypress/runner/cypress_runner.js:118316:18)
    at Async../node_modules/bluebird/js/release/async.js.Async._drainQueue (http://localhost:62304/__cypress/runner/cypress_runner.js:114928:16)
    at Async../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://localhost:62304/__cypress/runner/cypress_runner.js:114938:10)
    at Async.drainQueues (http://localhost:62304/__cypress/runner/cypress_runner.js:114812:14)]]></failure>
    </testcase>
  </testsuite>
</testsuites>